src/utils/downloadCsv.ts

```diff
@@ -0,0 +1,53 @@
+import type { CardItem } from '@/api/types';
+import { formatDate } from '@/utils/formatDate';
+
+function formatCsvCell(cell: string | number | null): string {
+  const str = String(cell);
```

Comment:
if you will have null in args - you will get:
`String(null) === 'null'`

use  const str = cell ? String(cell) : '';